Source/DafnyCore/Compilers/ExecutableBackend.cs

```diff
@@ -128,15 +128,17 @@ public abstract class ExecutableBackend : Plugins.IExecutableBackend {
     var dafny = where + "/Dafny.dll";
 
     var opt = Options;
-    var psi = PrepareProcessStartInfo("dotnet", opt.MainArgs
-      .Prepend("/compileTarget:cs")
-      .Prepend("/compile:4")
-      .Prepend("/compileVerbose:0")
-      .Prepend("/printVerifiedProceduresCount:0")
-      .Prepend("/noVerify")
-      .Prepend(targetFilename)
-      .Prepend(dafny));
-
+    var args = opt.MainArgs
+      .Prepend(targetFilename);
+    if (!verify) {
+      args = args.Prepend("--no-verify");
+    }
+    args = args
+      .Prepend("--target:cs")
+      .Prepend("run")
+      .Prepend(dafny);
```

Comment:
Nice for us to migrate internally to more use of the new CLI! Apart from fixing a bug, this seems worthwhile for its own sake.